acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/tls/ClientSideTlsAcceptanceTest.java

```diff
@@ -110,7 +115,7 @@ private Signer createSigner(
     final NodeConfiguration nodeConfig = new NodeConfigurationBuilder().build();
     final NodePorts nodePorts = new NodePorts(downstreamWeb3Port, UNUSED_WS_PORT);
 
-    final Signer signer = new Signer(builder.build(), nodeConfig, nodePorts);
+    signer = new Signer(builder.build(), nodeConfig, nodePorts);
```

Comment:
Think we probably also need to do the same changes for ServerSideTlsAcceptanceTest and ServerSideTlsCaClientAcceptanceTest